staticmap.go

```diff
@@ -256,12 +275,29 @@ func (r *StaticMapRequest) params() url.Values {
 }
 
 //StaticMap make a StaticMap API request
-func (c *Client) StaticMap(ctx context.Context, r *StaticMapRequest) (*http.Response, error) {
+func (c *Client) StaticMap(ctx context.Context, r *StaticMapRequest) (image.Image, error) {
 	if len(r.Markers) == 0 && r.Center == "" && r.Zoom == 0 {
 		return nil, errors.New("maps: Center & Zoom required if Markers empty")
 	}
 	if r.Size == "" {
 		return nil, errors.New("maps: Size empty")
 	}
-	return c.get(ctx, staticMapAPI, r)
+
+	resp, err := c.get(ctx, staticMapAPI, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != 200 {
+		if b, err := ioutil.ReadAll(resp.Body); err == nil {
+			return nil, fmt.Errorf("Static Map API returned: %s", string(b))
```

Comment:
this doesn't include the status code, if say the body doesn't: maybe include it?